src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -491,17 +491,17 @@ private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection
             int y = 0;
             switch(corner) {
                 case TOP_LEFT:
-                    x = screen.getWidth() - 90;
-                    y = screen.getHeight() - 90;
+                    x = screen.getWidth() - (screen.getWidth() - 10);
```

Comment:
`x - (x + 10)` is `10`. You can replace all those calculations with static values. I would also suggest extracting all  values `10` with a single constant called `MARGIN` or similar.